moped-editor/src/views/projects/projectView/ProjectComments.js

```diff
@@ -86,7 +86,8 @@ const useStyles = makeStyles(theme => ({
 const projectNoteTypes = ["", "Internal Note", "Status Update"];
 
 const ProjectComments = props => {
-  const { projectId } = useParams();
+  const isStatusEditModal = props.modal;
```

Comment:
This is just a different point of view, but I could see a world where two separate components named `ProjectComments` and something like `DashboardStatusEditModal` exist. Then, each component would reuse smaller components that have the rich text editor, etc. 

Not a huge concern here, but I'm just thinking back on how I made some of the project maps pull double-duty with shared logic and things broke down a bit.

Reply:
👍 definitely a pretty big refactor to extract out some of the mutation callbacks and what have you.

sort of related, i see our `<Page/>` component is getting wrapped up in this modal - which updates `<head>` with the page title "Project Notes" when the modal is open. 

i think we can drop `<Page/>` altogether, since afaik none of the other summary tabs use it - and it's kind of weird that the page title changes to "Project Notes" when you're viewing the project comments tab?

Reply:
I removed the Page component and Container and replaced it with CardContent to match the other Project tabs